packages/console/src/components/SubmitFormChangesActionBar/index.tsx

```diff
@@ -6,15 +8,18 @@ type Props = {
   isSubmitting: boolean;
   onSubmit: () => Promise<void>;
   onDiscard: () => void;
+  className?: string;
 };
 
-const SubmitFormChangesActionBar = ({ isOpen, isSubmitting, onSubmit, onDiscard }: Props) => {
-  if (!isOpen) {
-    return null;
-  }
-
-  return (
-    <div className={styles.container}>
+const SubmitFormChangesActionBar = ({
+  isOpen,
+  isSubmitting,
+  onSubmit,
+  onDiscard,
+  className,
+}: Props) => (
+  <div className={classNames(styles.container, isOpen ? styles.display : styles.hide, className)}>
```

Comment:
As the `.hide` styles can be merged into `.container` and provided by default, we only need to apply `.display` when the bar `isOpen`. We can also rename `.display` to `.active`.

So we can change it to:
```ts
classNames(styles.container, isOpen && styles.active)
```

Reply:
Cool, updated.